### PR TITLE
Add ranking score threshold

### LINKED ISSUE #641: [v1.9.0] Filter by ranking score
Related to https://github.com/meilisearch/integration-guides/issues/301

### Explanation of the feature

Introduction of a new search parameter to filter documents based on their ranking score

Usage: https://meilisearch.notion.site/Filter-by-score-usage-224a183ce7b24ca99b6a9a8da755668a?pvs=74

### TODO

- [ ] Ensure the `search` method accepts the `rankingScoreThreshold` search parameter
- [ ] Add tests

⚠️ Make PRs pointing to `bump-meilisearch-v1.9.0` and NOT `main`. Please do 1 PR for all of these changes, and not several.

### PR DESCRIPTION
**Pull request by the-sinner**
# Pull Request

## Related issue
Fixes #641 

## What does this PR do?
- Add ranking score threshold

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by the-sinner** [comment]
@curquiza do you want changes for #642 in the same PR or in a different PR ?

**Comment by curquiza** [comment]
Hello @the-sinner 
In a different PR, thank you very much 🙏

**Review by curquiza — APPROVED** [review]
thank you very much @the-sinner

File: src/Contracts/SearchQuery.php [hunk 1]
```diff
@@ -230,6 +238,7 @@ public function toArray(): array
             'attributesToSearchOn' => $this->attributesToSearchOn,
             'showRankingScore' => $this->showRankingScore,
             'showRankingScoreDetails' => $this->showRankingScoreDetails,
+            'rankingScoreThreshold' => $this->rankingScoreThreshold ?? null,
```
[the remark this answers is not in the document]
Comment (the-sinner) [reply src/Contracts/SearchQuery.php:241]: resolved in https://github.com/meilisearch/meilisearch-php/pull/645/commits/f63abad807ec4f2393f2e53463d952fd3f07ea07

**Pull request merged**